packages/jellyfish-api-core/src/category/account.ts

```diff
@@ -220,6 +220,20 @@ export class Account {
   ): Promise<AccountHistory[]> {
     return await this.client.call('listaccounthistory', [owner, options], 'number')
   }
+
+  /**
+   * Creates (and submits to local node and network) a transfer transaction from the wallet UTXOs to specfied account.
+   * The second optional argument (may be empty array) is an array of specific UTXOs to spend.
+   *
+   * @param {any} [payload]
+   * @param {UtxosToAccountUTXO[]} [options=[]]
+   * @param {string} [options.txid]
+   * @param {number} [options.vout]
+   * @return {Promise<string>}
+   */
+  async utxosToAccount (payload: any, options: UtxosToAccountUTXO[] = []): Promise<string> {
```

Comment:
you can give a proper name for 2nd param
use options only
see https://github.com/DeFiCh/jellyfish/blob/main/CONTRIBUTING.md#exported-functions-max-2-args-put-the-rest-into-an-options-object